pysteps/postprocessing/probmatching.py

```diff
@@ -259,6 +260,49 @@ def _get_error(scale):
     return shift, scale, R.reshape(shape)
 
 
+def resample_distributions(extrapolation_cascade, model_cascade, weight_extrapolation):
```

Comment:
mmmh I'm not sure about the new argument names, this method is rather agnostic with respect to what arrays are passed as long as they have the same shape, don't you think?

Reply:
True, but just a and b made it very unclear what the function is doing and what it needs as inputs from the `steps.py` code. What would you suggest?

Reply:
in the steps code you should leave the same variable names, but here I would keep it very general, something like`first_array` and `second_array` would do, no?

Reply:
That could work, though the method is very tailored to blending at the moment. Happy to change it otherwise.